src/resolvers.test.js

```diff
@@ -39,3 +42,38 @@ describe('ASM Resolver', () => {
     });
   });
 });
+
+describe('Cloudformation Output Resolver', () => {
+  it('returns cft output value', () => {
+    return cftResolver('testStack.foo').then(output => {
+      expect(output).toBe('bar');
+    });
+  });
+
+  it('calls to Cloudformation are cached', async () => {
+    const startTimesCalled = cft.timesCalled;
```

Comment:
These test methods are basically run in "parallel" in Node - any concerns about this global state not being correct in certain cases?